frontend/src/styles/Team.css

```diff
@@ -1,6 +1,5 @@
 #team{
     width: 90vw;
-    height: 80vh;
```

Comment:
I was just wondering why this line had to be removed? If it was to fix the issue of spacing between the team and tools component, I already corrected that in my last merge. I would suggest pulling the changes to make sure that it looks alright to you :)

Reply:
Thats why it was removed Palak. I had advised her to do so. Let me merge and see if that changes anything.